src/components/Chat.js

```diff
@@ -0,0 +1,214 @@
+import 'moment/locale/de';
```

Comment:
i think we do not need this here, as the momentFormat helper is setting the locale